src/components/transactions/walletTransactions/walletTransactions.test.js

```diff
@@ -68,8 +68,9 @@ describe('WalletTransactions Component', () => {
     transactionsCount: 1000,
     transaction: transactions[0],
     transactions,
-    loadTransactions: jest.fn(),
-    loadLastTransaction: jest.fn(),
+    getTransactions: jest.fn(),
+    getLastTransaction: jest.fn(),
+    updateAccountDelegateStats: jest.fn(),
```

Comment:
`updateAccountDelegateStats` should be removed here - it was removed in https://github.com/LiskHQ/lisk-hub/pull/2299